Core/Rewriters/MoqSyntaxFactory.cs

```diff
@@ -243,7 +243,7 @@ public static ExpressionStatementSyntax VerifyExpressionStatement (IdentifierNam
           MoqSyntaxFactory.VerifyExpression (identifierName));
     }
 
-    public static ExpressionSyntax VerifyExpression (IdentifierNameSyntax identifierName, ExpressionSyntax? expression = null, int? times = null)
+    public static ExpressionSyntax VerifyExpression (IdentifierNameSyntax identifierName, ExpressionSyntax? expression = null, int? times = null, ArgumentSyntax? options = null)
```

Comment:
The options name is a bit misleading, I guess this refers to the possibility to define "complex" expressions calculating the repeat count of the verification? Either way, the name should reflect what the parameter contains.